ruby-bionic/2.6.x/Dockerfile

```diff
@@ -65,15 +62,14 @@ RUN set -ex \
 	&& apt-get install -y --no-install-recommends $buildDeps \
 	&& rm -rf /var/lib/apt/lists/* \
 	\
-	&& wget -O ruby.tar.xz "https://cache.ruby-lang.org/pub/ruby/${RUBY_MAJOR%-rc}/ruby-$RUBY_VERSION.tar.xz" \
-	&& echo "$RUBY_DOWNLOAD_SHA256 *ruby.tar.xz" | sha256sum -c - \
```

Comment:
I wonder if we should figure out why we added the checksum in the first place.  I'm totally onboard with nixing it if it was just to do an integrity check, but if we added it for some sort of security compliance thing, we'll need to rework language somewhere, maybe?  I'm comfy with it now that we're scanning container images, though.

Reply:
I took that out since we're no longer pulling a source tarball from the ruby website, but instead checking out from git. I figured git has it's own integrity checks baked in and thought that would be good enough.

Should we run this by someone else? Security maybe?

Reply:
Looks like Mark W. put in that line in a few places I spot checked, they might know if it was just a checksum-the-download sort of thing or whether it was a "make sure nothing ever changes for compliance" sort of thing.  I'm fairly sure we're good to go, though!

Reply:
I just copied from the original base image but my guess is that since we aren't using a package manager some independent (i.e. outside of the website or now git) verification that the correct thing is downloaded is desired.  I'd be interested in what security has to say. Also, downloading  358MB vs 11MB... :O If we are changing things around maybe now is time to embrace https://github.com/rbenv/ruby-build?

Reply:
Earliest instance I could find is [here](https://github.com/socrata/shipyard/commit/215f7d51762712a61d7a7d5b60d7589268bd802d#diff-0c5e515e1d87d867a7d9d342d5af2a957df9d9a22cb83fef7f42ba7ee0a02df3R13). No obvious indication why.

I can give it a shot with ruby-build.